graphql_api/views.py

```diff
@@ -300,9 +302,11 @@ def context_value(self, request, *_):
         }
 
     def error_formatter(self, error, debug=False):
+        user = self.request.user
```

Comment:
if user is a "none" value wouldn't line 306 return a can't access on undefined error? 

maybe we could do a self.request.get("user",{})

Reply:
Not really! request.user is automatically populated by the AuthenticationMiddleware middleware, it exists in all requests as an instance of User or AnonymousUser, and fwiw we're not checking for user anywhere else in the view

Reply:
got it! Thanks for clarifying